packages/Card/src/Card.js

```diff
@@ -13,35 +13,35 @@ const propTypes = {
   /** Body content of the card. */
   children: PropTypes.node,
 
+  /** Sets className */
+  className: PropTypes.string,
+
   /** If the width of the card should span it's parent container (100%). */
   isFullWidth: PropTypes.bool,
 
-  /** If the height of the card should span the height of its children elements. */
-  isAutoHeight: PropTypes.bool,
-
   /** If the card is in an "active" or "selected" state. */
   isActive: PropTypes.bool,
 
   /** Size of the card (font size, min-height, padding, etc). */
-  size: PropTypes.oneOf(ShirtSizes.DEFAULT),
+  size: PropTypes.oneOf(["auto", ShirtSizes.SMALL, ShirtSizes.MEDIUM, ShirtSizes.LARGE]),
 };
 
 const defaultProps = {
   children: null,
-  size: ShirtSizes.MEDIUM,
+  className: null,
+  size: "auto",
   isFullWidth: false,
-  isAutoHeight: false,
   isActive: false,
 };
 
 function Card(props) {
-  const { children, size, isFullWidth, isAutoHeight, isActive } = props;
+  const { children, className, size, isFullWidth, isActive } = props;
```

Comment:
Instead of adding a `className` prop, we can keep this component agnostic of classes entirely, and allow other attributes (like `data-test-id`, etc) by simply adding `...moreProps` to the destructure list, then adding that same `...moreProps` to the `cardProps` object below.  This is a standard pattern in Paprika, you can find lots of examples in other components.